src/syscheckd/include/syscheck.h

```diff
@@ -112,6 +112,7 @@ typedef struct fim_txn_context_s {
 /* Flags to know if a directory/file's watcher has been removed */
 #define FIM_RT_HANDLE_CLOSED 0
 #define FIM_RT_HANDLE_OPEN 1
+#define REG_UNKNOWN 0x0000000C
```

Comment:
It would be good to include here a small comment about the need of `REG_UNKNOWN` and its value